content/news/017/index.md

```diff
@@ -30,7 +30,6 @@ Feel free to send PRs about your own projects!
 Table of contents:
 
 - [Rust Gamedev Meetup](#rust-gamedev-meetup)
-- [Rust GameDev Podcast](#rust-gamedev-podcast)
```

Comment:
This heading does exist, but it should be `[Rust GameDev Podcast #4](#rust-gamedev-podcast-4)`.